mario-redis-lock.gemspec

```diff
@@ -19,7 +19,7 @@ Gem::Specification.new do |spec|
   spec.test_files    = spec.files.grep(%r{^(test|spec|features)/})
   spec.require_paths = ["lib"]
 
-  spec.add_runtime_dependency 'redis', '~> 3', '>= 3.0.5' # Needed support for SET with EX, PX, NX, XX options: https://github.com/redis/redis-rb/pull/343
+  spec.add_runtime_dependency 'redis', '>= 3.0.5', '< 5' # Needed support for SET with EX, PX, NX, XX options: https://github.com/redis/redis-rb/pull/343
```

Comment:
why does the regis gem version need to be `< 5`?

Reply:
redis 5 has not been released yet. Compatibility may be broken in major upgrade.

For safety, should avoid installing on redis 5 until redis 5 release.